erigon-lib/kv/mdbx/kv_mdbx.go

```diff
@@ -1042,13 +1042,24 @@ func (tx *MdbxTx) IncrementSequence(bucket string, amount uint64) (uint64, error
 	return currentV, nil
 }
 
+func (tx *MdbxTx) ResetSequence(bucket string, newValue uint64) error {
```

Comment:
FYI: if we don't need support decrement - then can use native mdbx's sequence method

Reply:
ResetSequence is used in two places:
- truncate all tables; and reset the sequence to 0 -- probably don't need to reset sequence here, can just continue with whatever value is in there since it's txId.
- FillDBFromSnapshots -- i think it always increments, since it's executed in forward exec of snapshots stage.

So we should still be able to use the native mdbx's sequence method.